core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -514,6 +512,29 @@ protected ObjectMapper getJsonMapper() {
     return new ObjectMapper();
   }
 
+  /**
+   * Returns the default {@link Slf4jRequestLog} used. Separate method made available in order to
```

Comment:
not sure i understand the justification for the extra method. a subclass could just use `super` to access the original version of `getRequestLog`, right?

Reply:
Right, but then there would be no guarantee that the call to `getRequestLog()` would return a `Slf4jRequestLog`; instead, it would just be a `RequestLog`. This way, if you want to return something that is a `RequestLog` but isn't a `Slf4jRequestLog` by overriding `getRequestLog()`, you can--and if you want to access the default request log with the guarantee that it's a `Slf4jRequestLog`, you can too.